### PR TITLE
Add stub template for CF on Kubernetes WG

### PR DESCRIPTION
**Pull request by gcapizzi**
This PR proposes a working group dedicated to providing the Cloud Foundry developer experience on top of Kubernetes and its ecosystem.

### CODE REVIEW

File: toc/working-groups/cf-on-k8s.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Bring the ease and simplicity of the Cloud Foundry developer experience to Kubernetes.
+
+
+## Goals
+
+- End-user developers can deploy their applications through the same convenient flows provided by their existing Cloud Foundry environments, using their existing clients and tools (like the `cf` CLI or the CF Java client), relying on the same (V3) API.
+- End-user developers can take advantage of Cloud Foundry via Kubernetes-native APIs, using their existing tools and clients (e.g. `kubectl`, `client-go`).
+- End-user developers can smoothly migrate their workloads from their existing Cloud Foundry environments to Cloud Foundry on Kubernetes.
+- End-user operators have an robust way to install Cloud Foundry on their Kubernetes clusters.
```
Comment (tcdowney) [inline toc/working-groups/cf-on-k8s.md:13]:
```suggestion
- End-user operators have a robust way to install Cloud Foundry on their Kubernetes clusters.
```

Small typo

File: toc/working-groups/cf-on-k8s.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Bring the ease and simplicity of the Cloud Foundry developer experience to Kubernetes.
+
+
+## Goals
+
+- End-user developers can deploy their applications through the same convenient flows provided by their existing Cloud Foundry environments, using their existing clients and tools (like the `cf` CLI or the CF Java client), relying on the same (V3) API.
+- End-user developers can take advantage of Cloud Foundry via Kubernetes-native APIs, using their existing tools and clients (e.g. `kubectl`, `client-go`).
+- End-user developers can smoothly migrate their workloads from their existing Cloud Foundry environments to Cloud Foundry on Kubernetes.
+- End-user operators have a robust way to install Cloud Foundry on their Kubernetes clusters.
```
Comment (tcdowney) [inline toc/working-groups/cf-on-k8s.md:13]: I'm wondering if we can specify more what "robust" means here. I think it could imply something fully production-ready like cf-deployment which we're not equipped to develop or maintain. I'm wondering if that's actually a Non-Goal and our goal here is similar to the cf-for-k8s strategy of providing a deployment optimized for local development that others in the community can productionalize (dunno if this is a word 😅 ) if they want to.

cc @emalm for thoughts

Comment (emalm) [reply toc/working-groups/cf-on-k8s.md:13]: Yeah, we called out in the [Vision doc](https://docs.google.com/document/d/1rG814raI5UfGUsF_Ycrr8hKQMo1RH9TRMxuvkgHSdLg/edit) that "community reference deployment of CF on K8s should be optimized for initial, lightweight usage and experimentation", which aligns more to the last goal statement. Maybe it would make sense to rephrase this as something like "Vendors can base their own CF distributions on the core CF on K8s components"?

File: toc/working-groups/cf-on-k8s.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Develop a distribution of Cloud Foundry on Kubernetes that is easy to use, even without any Cloud Foundry-specific experience and works out-of-the-box.
+- Work with other Working Groups to make sure that the existing Cloud Foundry clients (e.g. the `cf` CLI, the CF Java client) work with Cloud Foundry on Kubernetes out-of-the-box.
+
+## Non-Goals
+
+- Provide full compatibility with the existing Cloud Foundry behaviour.
+
+## Proposed Membership
+
+Technical and Execution Lead: ?
```
Comment (emalm) [inline toc/working-groups/cf-on-k8s.md:30]:
```suggestion
Technical and Execution Lead: @gcapizzi 
```

**Comment by emalm** [comment]
@gcapizzi I pushed some revisions. PTAL!

**Comment by gcapizzi** [comment]
> @gcapizzi I pushed some revisions. PTAL!

LGTM!

**Review by emalm — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Review by emalm — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Comment by emalm** [comment]
Merging: a quorum of the TOC has approved and there has been no outstanding discussion for the past week.
